backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -734,4 +734,8 @@ SearchException invalidSingleValuedProjectionOnValueFieldInMultiValuedObjectFiel
 
 	@Message(id = ID_OFFSET + 157, value = "Unable to export the schema for '%1$s' index: %2$s" )
 	SearchException unableToExportSchema(String indexName, String message, @Cause IOException e);
+
+	@Message(id = ID_OFFSET + 158, value = "'%1$s' projections are only applicable within the root projection context. "
```

Comment:
I'm not sure "root projection context" is meaningful for all users... Maybe rather:

```suggestion
	@Message(id = ID_OFFSET + 158, value = "'%1$s' projections cannot be nested in an object projection. "
```